scss/_patterns_icons.scss

```diff
@@ -459,3 +470,99 @@
     }
   }
 }
+
+@mixin vf-p-icon-applications {
+  .p-icon--applications {
+    @extend %icon;
+    @include vf-icon-applications($color-mid-dark);
+
+    [class*='--dark'] &,
+    &.is-light {
+      @include vf-icon-applications($color-mid-x-light);
+    }
+  }
+}
+
+@mixin vf-p-icon-chevron-up {
```

Comment:
@lyubomir-popov This kind of icon seems pretty generic. Maybe we should add this one to core Vanilla?

We do use up chevron in places (like when opening contextual menu), but I guess we did that by rotating regular chevron instead.
Should we add it as a separate icon at all? Maybe `.p-icon--chevron-up` could use the same SVG as chevron but be rotated?